website/docs/browser-support.md

```diff
@@ -0,0 +1,131 @@
+- `dead`: browsers without official support or updates for 24 months. Right now it is `IE 10`, `IE_Mob 11`, `BlackBerry 10`, `BlackBerry 7`, `Samsung 4` and `OperaMobile 12.1`.
+- `last 2 versions`: the last 2 versions for -each- browser.
+  - `last 2 Chrome versions`: the last 2 versions of Chrome browser.
+  - `last 2 major versions` or `last 2 iOS major versions`: all minor/patch releases of last 2 major versions.
+- `node 10` and `node 10.4`: selects latest Node.js `10.x.x` or `10.4.x` release.
+  - `current node`: Node.js version used by Browserslist right now.
+  - `maintained node versions`: all Node.js versions, which are [still maintained](https://github.com/nodejs/Release) by Node.js Foundation.
+- `iOS 7`: the iOS browser version 7 directly.
+  - `Firefox > 20`: versions of Firefox newer than 20. `>=`, `<` and `<=` work too. It also works with Node.js.
+  - `ie 6-8`: selects an inclusive range of versions.
+  - `Firefox ESR`: the latest [Firefox Extended Support Release](https://support.mozilla.org/en-US/kb/choosing-firefox-update-channel).
+  - `PhantomJS 2.1` and `PhantomJS 1.9`: selects Safari versions similar to PhantomJS runtime.
+- `extends browserslist-config-mycompany`: take queries from `browserslist-config-mycompany` npm package.
+- `supports es6-module`: browsers with support for specific features. `es6-module` here is the `feat` parameter at the URL of the [Can I Use] page. A list of all available features can be found at [`caniuse-lite/data/features`](https://github.com/ben-eb/caniuse-lite/tree/master/data/regions).
+- `browserslist config`: the browsers defined in Browserslist config. Useful in Differential Serving to modify user’s config like `browserslist config and supports es6-module`.
+- `since 2015` or `last 2 years`: all versions released since year 2015 (also `since 2015-03` and `since 2015-03-10`).
+- `unreleased versions` or `unreleased Chrome versions`: alpha and beta versions.
+- `not ie <= 8`: exclude IE 8 and lower from previous queries.
+
+You can combine multiple queries with combiners. Start with a single query and then combine the queries to get the final list.
+
+An `or` combiner can use the keyword `or` as well as `,`. `last 1 version or > 1%` is equal to `last 1 version, > 1%`.
+
+An `and` combiner performs an intersection of all the previous queries: `last 1 version or chrome > 75 and > 1%` will select (`browser last version` or `Chrome since 76`) and `more than 1% marketshare`.
+
+A `not` combiner performs a relative complement: `> 0.5%, not dead` eliminates the browsers that are `dead` from the ones selected by `> 0.5%`. Obviously you can _not_ start with a `not` combiner, since there is no left-hand side query to combine it with. The left-hand is always resolved as `and` combiner even if `or` is used (this is an API implementation specificity).
+
+## Choosing a value {#choosing-a-value}
```

Comment:
I think this section is unnecessary, we should just link to the browserslist doc

Reply:
@chetstone Makes sense. I did make slight modifications to the explanation, but they're kind of interspersed in the copy-pasting. @slorber Do you think I should prune out some copy-pasted text?

Reply:
Not against adding more explainations, but not a direct and exhaustive copy/paste of the doc